specification/databricks/resource-manager/Microsoft.Databricks/preview/2022-10-01-preview/accessconnector.json

```diff
@@ -0,0 +1,485 @@
+          "200": {
+            "description": "OK - Returns an array of azure databricks accessConnectors.",
+            "schema": {
+              "$ref": "#/definitions/AccessConnectorListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "databricks.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Databricks/accessConnectors": {
+      "get": {
+        "tags": [
+          "AccessConnector"
+        ],
+        "operationId": "AccessConnectors_ListBySubscription",
+        "description": "Gets all the azure databricks accessConnectors within a subscription.",
+        "x-ms-examples": {
+          "Lists all the azure databricks accessConnectors within a subscription.": {
+            "$ref": "./examples/AccessConnectorsListBySubscriptionId.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of azure databricks accessConnectors.",
+            "schema": {
+              "$ref": "#/definitions/AccessConnectorListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "databricks.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AccessConnector": {
+      "type": "object",
+      "properties": {
+        "identity": {
+          "$ref": "#/definitions/IdentityData"
+        },
+        "properties": {
+          "description": "Azure Databricks accessConnector properties",
+          "$ref": "#/definitions/AccessConnectorProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "databricks.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Information about azure databricks accessConnector."
+    },
+    "AccessConnectorProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "Provisioning status of the accessConnector.",
+          "readOnly": true,
+          "enum": [
+            "Deleted",
+            "Failed",
+            "Succeeded"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "IdentityData": {
```

Comment:
>IdentityData [](http://example.com/codeflow?start=5&length=12)

Recommend using the ManagedServiceIdentity definition from v4 common types (managedIdentity.json) for this.